plugin/pkg/auth/authenticator/password/passwordfile/passwordfile.go

```
@@ -37,6 +40,21 @@ type userPasswordInfo struct {
 // NewCSV returns a PasswordAuthenticator, populated from a CSV file.
 // The CSV file must contain records in the format "password,username,useruid"
 func NewCSV(path string) (*PasswordAuthenticator, error) {
+	users, err := ReadCSV(path)
+	if err != nil {
+		return nil, err
+	}
+	authenticator := &PasswordAuthenticator{
+		path:  path,
+		users: users,
+	}
+	ready := make(chan bool)
+	go authenticator.AddFileWatcher(&ready)
```

Additionally, you're ignoring the error.